### ISSUE TITLE
Recommendation: Improve accessibility

### ISSUE THREAD

**Issue by melissamcewen** [Kennethrc2401/Warehouse-Capstone-Project#5]
Rubric:  The website includes HTML5 tags and any additional aria roles, labels, etc. so a screen reader is able to scan the website for a user.

There are a lot of great accessibility tools out there and I usually use Google Chrome's audit 
https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en

or this extension https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en 

These showed me some issues with the site:
* Missing alt tags on images (https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/links/all-links-must-contain-either-text-or-an-image-with-alt-text/)
* Missing labels on forms (someone with a screen reader couldn't figure out how to use search without this)
* The heading structure should basically be an outline of the content on the page in order and here it doesn't quite match that
![nav](https://user-images.githubusercontent.com/1251094/36652107-1da27a68-1a72-11e8-8b5e-c0fe67f8d70c.jpg)
I would even consider wrapping that footer menu in a nav element or giving it an aria label https://www.w3.org/WAI/tutorials/page-structure/regions/#navigation

I would also consider adding hover effects to links and the image menus http://uxmovement.com/buttons/why-your-links-need-a-hover-effect/

One note about these is these not only help people who use screen readers or other assistive technology, but they are also important for SEO (search engine optimization) so just a good idea to think about all around
